src/fiatExchanges/quotes/normalizeQuotes.ts

```diff
@@ -78,20 +79,42 @@ export function normalizeFiatConnectQuotes(
 
 export function normalizeExternalProviders(
   flow: CICOFlow,
-  input: FetchProvidersOutput[]
+  input: FetchProvidersOutput[],
+  digitalAsset: CiCoCurrency
 ): NormalizedQuote[] {
   const normalizedQuotes: NormalizedQuote[] = []
 
   input.forEach((provider) => {
     try {
-      if (provider.quote) {
-        // Sometimes the quote is an array and sometimes its a single quote
-        const quotes = Array.isArray(provider.quote) ? provider.quote : [provider.quote]
-        quotes.forEach((quote: RawProviderQuote | SimplexQuote) => {
-          const normalizedQuote = new ExternalQuote({ quote, provider, flow })
-          normalizedQuotes.push(normalizedQuote)
-        })
+      const quotes: (RawProviderQuote | SimplexQuote)[] = []
+      if (Array.isArray(provider.quote)) {
+        // If the quote object is an array, it may contain quotes, or be empty.
+        // If it is empty, it does not necessarily mean that the provider does not
+        // support transfers.
+        if (provider.quote.length) {
+          quotes.push(...provider.quote)
+        } else if (!provider.unavailable && !provider.restricted) {
+          // If no quotes are provided, but the provider is not marked as unavailable,
+          // this means the provider supports transfers, but does not give specific quote details.
+          // We construct a partial quote object for each payment method. N.B., this will never occur
+          // for Simplex quotes, which are a special case.
+          provider.paymentMethods.forEach((paymentMethod) =>
+            quotes.push({
+              digitalAsset,
```

Comment:
@jh2oman Now that `RawProviderQuote` includes _optional_ fields, rather than post-hoc constructing partial quote objects in the wallet, we can actually construct them in the cloud function instead. This would make the wallet code a bit cleaner (and prevent us from passing `digitalAsset` through here), with little(?) added complexity in the cloud function, I think.